server/src/test/java/org/opensearch/wlm/tracking/QueryGroupResourceUsageTrackerServiceTests.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.wlm.tracking;
```

Comment:
Package path should match the source code for code coverage report generation. `tracker` instead of `tracking`

Reply:
Good catch !